src/Player.Vm.Api/Features/Vms/VmService.cs

```diff
@@ -419,7 +404,7 @@ public async Task<VmMap[]> GetViewMapsAsync(Guid viewId, CancellationToken ct)
             var accessableMaps = new List<Domain.Models.VmMap>();
             foreach (var m in maps)
             {
-                if ((await _playerService.CanAccessTeamsAsync(m.TeamIds, ct)))
+                if ((await _playerService.CanAccessTeamsAsync(m.TeamIds, ct) || m.TeamIds.Count == 0))
```

Comment:
I think this reads that if the map has no team assigned to it, anyone in the system can access the map? Probably need to check CanManage on the user's primary team in this view if there are no teams assigned.